lib/perso.ml

```diff
@@ -1216,6 +1239,295 @@ let warning_use_has_parents_before_parent (fname, bp, ep) var r =
   |> !GWPARAM.syslog `LOG_WARNING;
   r
 
+let cousins_t = ref None
+let cousins_dates_t = ref None
+
+let max_l1_l2_aux env =
+  let max_l1 = match get_env "max_anc_level" env with Vint i -> i | _ -> 5 in
+  let max_l2 = match get_env "max_desc_level" env with Vint i -> i | _ -> 5 in
+  (max_l1 - 1, (2 * max_l1) + max_l2 + 20)
```

Comment:
same here, what is the meaning of the `20`

Reply:
good catch: that +20 was on a temporary little commit I did to work more a night, it was a hard limit added before calculus of extremum of the table were good (the l2 descendant limit was too short, so I added an very arbitrary one because it was missing some descendants when l1/l2 were very low). It's a problem we had 2 or 3 months ago at the start, and I was thinking yesterday to remove it or find something better while rebasing/cleaning the whole branch.

Reply:
I think its ok to have a hard coded value, but it is important to have a name on it to better understand